easybuild/easyblocks/julia.py

```diff
@@ -56,8 +56,9 @@ def extra_options(extra_vars=None):
     def get_environment_folder(self):
         env_path = ''
 
-        hostname = socket.gethostname()
-        hostname_short = ''.join(c for c in hostname if not c.isdigit())
+        #hostname = socket.gethostname()
+        #hostname_short = ''.join(c for c in hostname if not c.isdigit())
```

Comment:
I suggest to omit the previous implementation from the code, we can refer to git if we need old versions.